content/comm/abstract-review/index.en.md

```diff
@@ -29,5 +29,33 @@ in particular before big conference deadlines,
 
 ### Coordination
 
-The current volunteer in charge of the system uses a Trello board for tracking 
-review requests.
\ No newline at end of file
+The requests and volunteers are tracked in AirTable: https://airtable.com/appJadVolZxoDGSIK?
+
+Current volunteers (Nic and Saya) alternate managing reviews on a monthly basis with Nic on even numbered months and Saya on odd-numbered months.
+
+## Handling an incoming request
+
+Incoming requests trigger an automated message to be posted in the #team-abstract_review channel in R-Ladies Slack.
+
+The review handler should:
+
+1. Check the request and the submission deadline, preferred reviewer language and gender, comments made by requestor, and that any attached documents are accessible
+2. Select a relevant reviewer from the list of volunteers, incrementing the value in the "reviews requested" column by 1.  If possible, try to use volunteers who haven't done too many previous reviews.
+3. Email the potential reviewer using the template below or similar
```

Comment:
```suggestion
3. Email 3 potential reviewers using the template below or similar
```

Perhaps if we have 3 reviewers by default, there's less onus on us to check in as to whether they've done the review, as it's more likely at least 1 will